docs/docs/cli/reference.md

```diff
@@ -95,7 +95,7 @@ airy create [flags]
 
 ```
   -h, --help              help for create
-      --provider string   One of the supported providers (aws|local). Default is aws
+      --provider string   One of the supported providers (aws|local|minikube). Default is aws (default "local")
```

Comment:
Isn't the `minikube` provider the equivalent of `local`?

Or, to put it differently? How do we use `local` provider without minikube?

Reply:
https://github.com/airyhq/airy/pull/1151/files#diff-bf9f1ed02d14293e3d603f8a142bb5817e45f30da87a16c5a4f09f1484751fcaR30

The `local` provider is only temporary until we do the minikube PR. It reflects the current cli api, but makes it explicit as a provider to allow for the restructuring.